console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/shared/util/GwtKapuaJobModelConverter.java

```diff
@@ -156,6 +156,9 @@ public static JobTargetQuery convertJobTargetQuery(GwtJobTargetQuery gwtJobTarge
             jobTargetQuery.setOffset(loadConfig.getOffset());
 
             String sortField = StringUtils.isEmpty(loadConfig.getSortField()) ? JobTargetPredicates.ENTITY_ID : loadConfig.getSortField();
+            if (sortField.equals("displayName")) {
+                sortField = JobTargetPredicates.JOB_TARGET_ID;
```

Comment:
Sorting will never work on the display name. Just disable it from the column `ColumnConfig`